operation.js

```
@@ -1,47 +1,62 @@
-const { Client } = require('pg');
+const { Client } = require("pg");
 
 // Database connection details
 const dbConfig = {
-    user: 'user',
-    host: '127.0.0.1',
-    database: 'citus',
-    password: 'password',
-    port: 5432,
+  user: "user",
+  host: "127.0.0.1",
+  database: "citus",
+  password: "password",
+  port: 5432,
 };
 
 async function connectToDb() {
-    const client = new Client(dbConfig);
-    await client.connect();
-    return client;
+  const client = new Client(dbConfig);
+  await client.connect();
+  return client;
 }
 
 async function insertData(vin, lat, long, speed) {
-    const client = await connectToDb();
-    await client.query('INSERT INTO vehicle_tracking (vin, latitude, longitude, speed) VALUES ($1, $2, $3, $4)', 
-        [vin, lat, long, speed]);
-    await client.end();
```

Empty `catch` block: swallowing errors silently makes debugging difficult and can hide critical runtime issues. Consider at least logging the error with `console.error(e)` or handling it appropriately.